### DISCUSSION TITLE
Domain Stats

### DISCUSSION THREAD

**Discussion by secraven** [Security-Onion-Solutions/securityonion#1771] [Unsupported Versions] [answered]
Where can I find domain stats information in Kibana? The older version had its own dashboard. I can't find a similar dashboard in the version.

Thank You!

https://docs.securityonion.net/en/2.3/domainstats.html?highlight=domain%20stats

**Comment by weslambert** [comment, marked as answer]
We haven't integrated full support for domainstats in Security Onion 2 at this time.

**Reply by secraven** [reply]
Thank you for the update!  Loving the new version.  Looking forward to your continued progress!
